.github/workflows/develop.yml

```diff
@@ -0,0 +1,24 @@
+name: dev
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: "11.0.15"
```

Comment:
Here we can't use `java-version: "11"` because it fails with the following error:

```
Error: Hash of java.rmi (3395a0342064c807f90dfa586cd5f024627336ea39cd1f7460b2247b6bfa333c) differs to expected hash (6ecd97f72f1d0e2049cb8c36297444f6d5b0baf535726fb3d9587829cb443243) recorded in java.base
```